core/src/main/resources/error/delta-error-classes.json

```diff
@@ -1320,7 +1320,8 @@
   },
   "DELTA_UNKNOWN_CONFIGURATION" : {
     "message" : [
-      "Unknown configuration was specified: <config>"
+      "Unknown configuration was specified: <config>",
+      "To disable this check, set allowArbitraryProperties.enabled=true in the Spark session configuration."
```

Comment:
Could you provide the full config key?

Reply:
You can see how we use a config key in this file here: https://github.com/delta-io/delta/blob/12debe9b4e058a12e6f8f1ba66453ff179a3fe50/core/src/main/resources/error/delta-error-classes.json#L1435

Reply:
Will do this in a follow-up PR

Reply:
SGTM